buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -1151,13 +1151,13 @@ private void createConfiguration() {
         } else {
             baseConfig.put("script.max_compilations_rate", "2048/1m");
         }
-        if (getVersion().getMajor() >= 6) {
+        if (getVersion().onOrAfter("6.0.0")) {
             baseConfig.put("cluster.routing.allocation.disk.watermark.flood_stage", "1b");
         }
         // Temporarily disable the real memory usage circuit breaker. It depends on real memory usage which we have no full control
         // over and the REST client will not retry on circuit breaking exceptions yet (see #31986 for details). Once the REST client
         // can retry on circuit breaking exceptions, we can revert again to the default configuration.
-        if (getVersion().getMajor() >= 7) {
+        if (getVersion().onOrAfter("7.0.0")) {
```

Comment:
nit:
```suggestion
        if (getVersion().onOrAfter(LegacyESVersion.V_7_0_0)) {
```

Reply:
build-tools has no dependency on the `server` module so `LegacyESVersion` is not available in the `buildSrc` module.